package.json

```diff
@@ -68,7 +68,7 @@
     "@elastic/elasticsearch-v7.17.0": "npm:@elastic/elasticsearch@7.17.0",
     "@elastic/elasticsearch-v7.9.0": "npm:@elastic/elasticsearch@7.9.0",
     "@google-cloud/pubsub": "3.3.0",
-    "@google-cloud/storage": "^5.16.1",
+    "@google-cloud/storage": "^6.9.4",
```

Comment:
IMO there is no reason to cover both version 5 and 6 in our tests, because the only difference or better said "breaking change" was that `resumable: true` is now default.